lib/plugin.js

```diff
@@ -7,6 +7,9 @@ import Vuetify from 'vuetify'
 
 Vue.use(Vuetify)
 
-export default ({ app }) => {
-  app.vuetify = new Vuetify(<%= JSON.stringify(options.vuetifyOptions, null, 2) %>)
+export default (ctx) => {
+  const vuetify = new Vuetify(<%= JSON.stringify(options.vuetifyOptions, null, 2) %>)
+
+  ctx.$vuetify = vuetify
+  ctx.app.vuetify = vuetify;
```

Comment:
It could be better to use inject instead of directly attaching to app.

Reply:
With `inject`:
```
TypeError
Cannot set property $vuetify of #<Vue> which has only a getter
```

Reply:
Also got this error using inject.
I already have done this PR locally, but then figured out that I needed to provide typings for typescript, but that `$vuetify` on Vue instances has `any` type around Vuetify typings, I'll help them to make it better defined

Reply:
Hmm OK. Seems vuetify plugin also self registers!

Reply:
@pi0 Yes, but their register is the Vuetify installed instance, meanwhile `app.vuetify` & `ctx.$vuetify` are options. But anyway I tried locally and succeed to register a method helper that switch the dark mode in a plugin when `ctx.$vuetify` is set. So I think everything is good as it's now. Will work later on types.